packages/dds/map/src/localValues.ts

```diff
@@ -80,14 +85,19 @@ export class PlainLocalValue implements ILocalValue {
 	/**
 	 * {@inheritDoc ILocalValue.makeSerialized}
 	 */
-	public makeSerialized(serializer: IFluidSerializer, bind: IFluidHandle): ISerializedValue {
+	public makeSerialized(
+		serializer: IFluidSerializer,
+		bind: IFluidHandle,
+		attribution?: AttributionKey | number,
+	): ISerializedValue {
 		// Stringify to convert to the serialized handle values - and then parse in order to create
 		// a POJO for the op
 		const value = serializeHandles(this.value, serializer, bind);
 
 		return {
 			type: this.type,
 			value,
+			attribution: JSON.stringify(attribution),
```

Comment:
this will cause unnecessary double stringification. we should retain the `AttributionKey` type here and only parse once (fix wherever you're parsing this as well)

Reply:
ah. i see there's extra weird parsing logic in summarization. not ideal that we stringify/parse a lot, but since this at least consistent, I think it's fine for now.